hazelcast/src/test/java/com/hazelcast/map/impl/mapstore/TestEntryLoader.java

```diff
@@ -21,22 +21,26 @@
 import java.util.Collection;
 import java.util.HashMap;
 import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.ConcurrentHashMap;
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.atomic.AtomicInteger;
 
 public class TestEntryLoader implements EntryLoader<String, String> {
 
     static final String NULL_RETURNING_KEY = "nullReturningKey";
 
-    private Map<String, Record> records = new HashMap<>();
+    private Map<String, Record> records = new ConcurrentHashMap<>();
     private AtomicInteger loadedEntryCount = new AtomicInteger();
     private AtomicInteger loadAllCallCount = new AtomicInteger();
     private AtomicInteger loadCallCount = new AtomicInteger();
+    private Set<String> loadUniqueKeys = ConcurrentHashMap.newKeySet();
```

Comment:
Minor: we can make this field `final` (as well as other fields in this class).

Reply:
Made the fields final.